src/Agent.Worker/TestResults/ResultsCommands.cs

```diff
@@ -0,0 +1,321 @@
+﻿using Microsoft.VisualStudio.Services.Agent.Util;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Globalization;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
+{
+    public class ResultsCommands : AgentService, ICommandExtension
+    {
+        public Type ExtensionType
+        {
+            get
+            {
+                return typeof(ICommandExtension);
+            }
+        }
+
+        public String CommandArea
```

Comment:
Consider: We generally use built-in type for primitive types. (string instead of String. int instead of Int32)